homeassistant/components/iotty/switch.py

```diff
@@ -0,0 +1,92 @@
+"""Turn the entity on."""
+import logging
+from typing import Any
+
+from iottycloud.lightswitch import LightSwitch
+from iottycloud.verbs import LS_DEVICE_TYPE_UID
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .api import IottyProxy
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IottyLightSwitch(SwitchEntity):
```

Comment:
For new integrations, we require the following:
- Support for [the new entity naming](https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations)
- Services must be [translatable](https://developers.home-assistant.io/blog/2023/07/11/translating-services)
- Entities and their attributes should be [translatable](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/)

Please adopt your code to add support for it.

Reply:
Thanks. Will include it in next commits.